packages/jellyfish-transaction/src/script/dftx/dftx_token.ts

```diff
@@ -117,3 +119,55 @@ export class CTokenUpdateAny extends ComposableBuffer<TokenUpdateAny> {
     ]
   }
 }
+
+/**
+ * Known as "std::variant<CScript>" in cpp.
+ */
+export type VariantType = 0
+interface VariantScript {
```

Comment:
```suggestion
export type VariantType = 0

export interface VariantScript {
```

missing export?

Reply:
Do you mind describing the interface `VariantScript`?

Reply:
Sure, it's created to handle [this](https://github.com/DeFiCh/ain/blob/master/src/masternodes/mn_checks.h#L337)


When there's an empty context: [0x0000000000](https://github.com/JellyfishSDK/jellyfish/pull/1929/files#diff-0586520016a5b57f3f7a2758934668f8654b63d05bd4a0a85aae02a593f57e0eR51)


```
std::variant 00000000
empty script 00
``` 

and when there's a context: [0x00000000160014ad54d71e8681e0c990349070cbd17a5c567a9b9e ](https://github.com/JellyfishSDK/jellyfish/pull/1929/files#diff-0586520016a5b57f3f7a2758934668f8654b63d05bd4a0a85aae02a593f57e0eR76)
```
std::variant 00000000
script  160014ad54d71e8681e0c990349070cbd17a5c567a9b9e
```

Thanks to @canonbrother for helping out here! 🙏